grad_buymenu/definitions/suppliesBlufor.sqf

```diff
@@ -115,6 +118,7 @@ _m109 = [
       clearItemCargoGlobal (_this select 0);
       clearBackpackCargoGlobal (_this select 0);
       clearMagazineCargoGlobal (_this select 0);
+	  (_this select 0) disableTIEquipment true;
```

Comment:
ace coding guideline kompatibel wäre spaces. ka was ich eingestellt hab @home. vmtl in atom spaces und in sublime tabs :trollface: